staging/src/k8s.io/apiserver/pkg/endpoints/handlers/get.go

```diff
@@ -99,6 +100,14 @@ func GetResource(r rest.Getter, scope *RequestScope) http.HandlerFunc {
 						return nil, errors.NewBadRequest("the export parameter, deprecated since v1.14, is no longer supported")
 					}
 				}
+				// resourceVersion should be "" or int
+				if len(values["resourceVersion"]) > 0 {
+					_, err := strconv.ParseUint(values["resourceVersion"][0], 10, 64)
```

Comment:
resourceVersion should be an opaque string, it happened to be 'int' with the current use of etcd as datastore.

Reply:
I just checked the type definition. You are right. 
So does this mean we should not fix #101350? @yliaog 

https://kubernetes.io/docs/reference/using-api/api-concepts/#resource-versions
> Resource versions must be treated as opaque by clients and passed unmodified back to the server.

https://github.com/kubernetes/kubernetes/blob/ea0764452222146c47ec826977f49d7001b0ea8c/staging/src/k8s.io/apimachinery/pkg/apis/meta/internalversion/types.go#L47-L50

Reply:
Only one datastore type is supported now:  etcd3.

>       --storage-backend string                   The storage backend for persistence. Options: 'etcd3' (default).